CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### New features and optimizations
 - Added public-facing official Dockerfile for NeMo-Aligner
+- Added the ability to do multi-epoch (cfg.max_epochs > 1) training for reward models, DPO, and PPO
```

Comment:
Is it really RM-specific? If not could be "supevised methods" rather than "reward models" (assuming it also includes SFT / SteerLM)

Reply:
Eh, close enough

Reply:
Unresolving since this hasn't been addressed as far as I can tell.

Reply:
Fixed now